node/tests/GlideClusterClient.test.ts

```diff
@@ -68,9 +69,33 @@ describe("GlideClusterClient", () => {
         cluster = clusterAddresses
             ? await ValkeyCluster.initFromExistingCluster(
                   parseEndpoints(clusterAddresses),
+                  async (addresses: [string, number][]) => {
```

Comment:
re-use the function

Reply:
Updated. For cluster client, we have passed a second parameter called as clusterMode(true for cluster).